UKHomeOffice/nmsw-ui
# NMSW-394 Update error messages

# Ticket

NMSW-394

----

## AC

UCD has reviewed all error messages and need to update app & tests to match. We're switching to 'harsher' validation error messages. So 'Enter a password that does not contain spaces' is changing to 'Password must not contain spaces'. This is in order to align more closely with COP and E@B error messages.

----

## To test

- Open spreadsheet attached to the ticket and scroll down to the current NMSW error messages
- Run app
- Compare the error messages marked as 'incorrect' on the spread sheet to the error messages on the app
- > The error messages in the app should match the new error messages in the table on the spreadsheet
- Check FAL 5 error messages for invalid and max characters (see note below)

----

## Notes

- **I have had to make changes to the FAL 5 error mapping due to a BE refactor - will need to check FAL 5 errors for invalid characters and max characters**
